### DISCUSSION TITLE
locale

### DISCUSSION THREAD

**Discussion by invaders-xx** [saade/filament-fullcalendar#42] [Q&A] [answered]
Why do you set locale separately in fullcalendar.blade.php as it is already set in getConfig?

**Comment by saade** [comment, marked as answer]
Because by default, it uses the Laravel locale config which is incompatible with the fullcalendar locale.

Laravel: `pt_BR`
Fullcalendar: `pt-br`

I'm not setting separately, only formatting the locale that comes from getConfig =)
